uk.co.compucorp.civicrm.contactactionsmenu/contactactionsmenu.civix.php

```diff
@@ -0,0 +1,445 @@
+<?php
+
+// AUTO-GENERATED FILE -- Civix may overwrite any changes made to this file
+
+/**
+ * The ExtensionUtil class provides small stubs for accessing resources of this
+ * extension.
+ */
+class CRM_Contactactionsmenu_ExtensionUtil {
```

Comment:
based on the namespace update I mentioned on another comment, this class name will need to be updated too